delivery/media/watermarking/lib/media-delivery-watermarking.js

```diff
@@ -0,0 +1,2134 @@
+import { WritableStream } from "streams";
+
+import { crypto, pem2ab } from "crypto";
+
+import { TextEncoder as TextEncoder$1, base64url, base16, atob } from "encoding";
+
+import { logger } from "log";
+
+import { IrdetoAlgorithm } from "irdeto-algorithm.js";
+
+import { httpRequest } from "http-request";
+
+var TokenType, KeysType;
+
+!function(TokenType) {
+    TokenType.CWT = "CWT", TokenType.JWT = "JWT";
+}(TokenType || (TokenType = {})), function(KeysType) {
+    KeysType[KeysType.BASE64 = 0] = "BASE64", KeysType[KeysType.HEX = 1] = "HEX";
+}(KeysType || (KeysType = {}));
+
+"undefined" != typeof globalThis ? globalThis : "undefined" != typeof window ? window : "undefined" != typeof global ? global : "undefined" != typeof self && self;
+
+/*!
+ * range-parser
+ * Copyright(c) 2012-2014 TJ Holowaychuk
+ * Copyright(c) 2015-2016 Douglas Christopher Wilson
+ * MIT Licensed
+ */ var rangeParser_1 = function(size, str, options) {
```

Comment:
Don't you [need to include the full license and copyright notice](https://snyk.io/learn/what-is-mit-license/#:~:text=The%20MIT%20license%20gives%20users,to%20suit%20their%20own%20needs.)?

Reply:
@evan-hughes so should i include MIT license file in the lib folder along with *.d.ts and *.js file? Or should i copy the entire the content and place it at the above place. The modules internally used cbor-x and range-header npm modules as 3p library which have MIT license.